reference/tools/env/virtualbuildenv.rst

```diff
@@ -67,6 +67,10 @@ will create the following files:
 
     To create ``.ps1`` files required for Powershell it is necessary to set to True the following conf: ``tools.env.virtualenv:powershell``.
 
+.. note::
+
+    Disabling the automatic generation of these files for consumers can be achieved by passing ``--env-generation=false`` to the ``conan install`` command.
```

Comment:
I'll add that suppressing these files means that some things won't work in some cases, and this should only be done if you are sure you don't need tool_requires and that stuff

I'd also add that this is experimental and might change

Reply:
Good call about the experimental status